logstash-core/lib/logstash/modules/kibana_config.rb

```diff
@@ -19,6 +19,7 @@ def initialize(modul, settings)
     @name = modul.module_name
     @settings = settings
     @index_name = settings.fetch("dashboards.kibana_index", ".kibana")
+    @metrics_max_buckets = settings.fetch("dashboards.metrics_max_buckets", 60000)
```

Comment:
We should use the formula that Nic proposed as its events per second - 24 hours is 86400 seconds = 86400 buckets